docs/src/tutorial/example-data.rst

```diff
@@ -0,0 +1,99 @@
+Run using example data
+======================
+
+The aim of this first tutorial is to introduce our SARS-CoV-2 workflow.
+To do this, we will run the workflow using a small set of reference data which we provide.
+This tutorial leads on to subsequent tutorials where we will walk through more complex scenarios.
+
+.. contents:: Table of Contents
+   :local:
+
+Prerequisites
+-------------
+
+1. :doc:`setup`. These instructions will install all of the software you need to complete this tutorial and others.
+
+Setup
+-----
+
+1. Activate the ``nextstrain`` conda environment:
```

Comment:
I'm a bit lost -- is this needed because the canonical docker-runtime flow would be to `git clone` etc from within the docker container (i.e. within `nextstrain shell .`)? If this is so, then I'd advocate for "Create a conda environment named nextstrain and install the Nextstrain CLI:" to also install `git`.

Reply:
`git` is both included in Native install directions and [installed in the Docker image](https://github.com/nextstrain/docker-base/blob/5e61a58d0e0b6b49cd99204b449eb82204c1ed7e/Dockerfile#L20), so I think that's not an issue.

I don't think this should block merge of the PR, so I've written up in more detail here: nextstrain/docs.nextstrain.org#108

Reply:
I got confused when writing this. Closed the issue and resolving this conversation.